src/folio_migration_tools/migration_tasks/orders_transformer.py

```diff
@@ -166,33 +166,15 @@ def process_single_file(self, filename):
                         record, f"row {idx}", FOLIONamespaces.orders, True
                     )
 
-                    # Handle merging and storage
-                    if not self.current_folio_record:
-                        self.current_folio_record = folio_rec
-                    if (
-                        self.current_folio_record
-                        and folio_rec["id"]
-                        and folio_rec["id"] != self.current_folio_record.get("id", "")
-                    ):
-                        # Writes record to file
-                        Helper.write_to_file(results_file, self.current_folio_record)
-                        self.mapper.migration_report.add_general_statistics(
-                            "Orders written to disk"
-                        )
-                        self.current_folio_record = folio_rec
+                    # Add notes
+                    self.mapper.notes_mapper.map_notes(
+                        record,
+                        legacy_id,
+                        folio_rec["id"],
+                        FOLIONamespaces.organizations,
```

Comment:
Should this be `orders` rather than `organizations`?

Reply:
So true! Thank you.